hack/00-osd-managed-cluster-config-integration.yaml.tmpl

```diff
@@ -18914,6 +19294,307 @@ objects:
       metadata:
         name: ocm-agent-operator
         namespace: openshift-ocm-agent-operator
+- apiVersion: hive.openshift.io/v1
```

Comment:
This is what i meant by the comment above. This selectorSyncSet will be synced onto the classic OSD clusters as well.

Reply:
I added a pretty hacky kludge to get the splunk-audit-exporter only installed on the hypershift clusters, but as that includes some changes to the python script, I guess that should not be merged just like that.